lib/engine_socketio.js

```diff
@@ -118,20 +118,21 @@ SocketIoEngine.prototype.step = function (requestSpec, ee) {
     return engineUtil.createLoopWithCount(requestSpec.count || -1, steps);
   }
 
-  let f = function(context, callback, socketio) {
+  let f = function(context, callback) {
```

Comment:
Removed socketio parameter.

Socket is now taken from context.sockets[requestSpec.emit.namespace] at line `132`.